modules/social_features/social_group/social_group.group.permissions.yml

```diff
@@ -0,0 +1,3 @@
+view group stream page:
```

Comment:
Does this need an update hook to ensure that users that now have the 'view group stream page' permission get the 'view group stream page' group permission?

Reply:
We set group permission in config files. And this one _view group stream page_ was already there, so with this file it will just appear in Group Permissions UI. 
But thanks for a notice!

Reply:
I don't think we can write an update for that. You can't make assumptions about this permission and what it means per group type :(